src/NzbDrone.Core/DecisionEngine/Specifications/RssSync/AvailabilitySpecification.cs

```diff
@@ -28,8 +28,27 @@ public Decision IsSatisfiedBy(RemoteMovie subject, SearchCriteriaBase searchCrit
                     return Decision.Accept();
                 }
             }
-            //this should then become a check if subject.Movie.Status >= subject.Movie.MinimumAvailability
-            if (subject.Movie.Status != MovieStatusType.Released)
+
+            MovieStatusType minAvail;
+            int x = 0;
+            bool result = Int32.TryParse(subject.Movie.Minimumavailability, out x);
```

Comment:
so I want to fix this up using an enum... it will make the code alot nicer.. The thing is, everything is built off from the enum:

namespace NzbDrone.Core.Tv
{
    public enum MovieStatusType
    {
        TBA = 0, //Nothing yet announced, only rumors, but still IMDb page
        Announced = 1, //AirDate is announced but hasn't been reached yet
        InCinemas = 2, // past Cinema release date but not physical release date
        Released = 3 //passed physical/web release date
    }
}

and I used that as much as I knew how or could.. but with respect to the values coming in from the fields.. i used a string... I noticed the profiles doing something else, so I planned to go back and try and fix that up. Im looking into this now.